sqlalchemy_redshift/dialect.py

```diff
@@ -882,6 +866,49 @@ def _get_all_constraint_info(self, connection, **kw):
         return all_constraints
 
 
+class PsycopgRedshiftDialectMixin(RedshiftDialectMixin):
+    """
+    Define Psycopg specific behavior.
+
+    Most public methods are overrides of the underlying interfaces defined in
+    :class:`~sqlalchemy.engine.interfaces.Dialect` and
+    :class:`~sqlalchemy.engine.Inspector`.
+    """
+    def create_connect_args(self, *args, **kwargs):
+        """
+        Build DB-API compatible connection arguments.
+
+        Overrides interface
+        :meth:`~sqlalchemy.engine.interfaces.Dialect.create_connect_args`.
+        """
+        default_args = {
+            'sslmode': 'verify-full',
+            'sslrootcert': pkg_resources.resource_filename(
+                __name__,
+                'redshift-ca-bundle.crt'
+            ),
+        }
+        cargs, cparams = (
+            super(PsycopgRedshiftDialectMixin, self).create_connect_args(
+                *args, **kwargs
+            )
+        )
+        default_args.update(cparams)
+        return cargs, default_args
+
+
+class Psycopg2RedshiftDialect(
```

Comment:
I'd like to following existing naming conventions in SQLAlchemy where possible. From the postgresql dialect code, it looks like this would be called `RedshiftDialect_psycopg2`. Does that seem like a reasonable choice here?

Reply:
Yes, sounds good. I will make this change

Reply:
addressed in 29d5704d